### PR TITLE
Feature/position select field

### PR DESCRIPTION
**Pull request by alagregor**
[Reference JIRA 4240](https://jira.greenpeace.org/browse/PLANET-4240)

### CODE REVIEW

File: classes/controller/class-enform-fields-list-table.php [hunk 1]
```diff
@@ -65,6 +65,8 @@ public function prepare_items() {
 			}
 		}
 
+		print_r($supporter_fields);
```
Comment (sagarsdeshmukh) [inline classes/controller/class-enform-fields-list-table.php]: Hi @alagregor - Please remove debugging code.

**Comment by sagarsdeshmukh** [comment]
As per JIRA ticket details, on select of other option from dropdown, user need to add text input. Is this requirement still valid? other than that PR looks good to me 👍 

> Michale Austin mentioned in the P4-en-integration channel that the field should be a "Select with input" field, which means there should be a text field below the dropdown. This might require some design from William Morris-Julien. If we need this field, should the textbox be shown ONLY when the "Other" option from dropdown is selected? If so then this will require the field to always have an "Other" option

File: classes/controller/class-enform-fields-list-table.php (hunk 1 above)
Comment (alagregor) [reply classes/controller/class-enform-fields-list-table.php]: Removed debugging code. Please review.

**Comment by alagregor** [comment]
Regarding this additional input field, for now it is not needed, this is arranged with Marija Tomac and we need this drop down as soon as possible on live page. If this input field is however needed, can be added to separate task and arranged after this change. Thanks.

**Comment by comzeradd** [comment]
Merged at #31

**Pull request closed without merging**